bindings/nodejs/lib/secret_manager/secret-manager-method-handler.ts

```diff
@@ -1,7 +1,11 @@
 // Copyright 2021-2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-import { callSecretManagerMethodAsync, createSecretManager } from '../bindings';
+import {
+    callSecretManagerMethodAsync,
+    createSecretManager,
+    migrateStrongholdSnapshotV2ToV3,
+} from '../bindings';
```

Comment:
Looks like you need to export migrateStrongholdSnapshotV2ToV3 also from bindings/wasm/lib/bindings.ts